core/src/com/unciv/ui/victoryscreen/VictoryScreen.kt

```diff
@@ -223,8 +224,12 @@ class VictoryScreen(val worldScreen: WorldScreen) : PickerScreen() {
 
         for (category in RankingType.values()) {
             val headers = Table().apply { defaults().pad(5f) }
-            headers.add(category.name.replace('_', ' ').toLabel()).row()
-            headers.addSeparator().fillX()
+            val textAndIcon = Table().apply { defaults().pad(5f) }
+            val columnImage = getRankingColumnIcon(category)
+            if (columnImage != null) textAndIcon.add(columnImage).center().size(Constants.defaultFontSize.toFloat() * 0.75f).padRight(0f).padTop(0f)
+            textAndIcon.add(category.name.replace('_', ' ').toLabel()).pad(5f).row()
+            headers.add(textAndIcon)
+            headers.addSeparator().size(0f, -1f).fillX()
```

Comment:
:eyes:  that works? What does it do? Zero width, negative height? And fillX is already set by addSeparator?

Reply:
This adds the image with some trial-and-error sizing/padding to get the new icons to match the same size/spacing as the existing Stat icons.